trollmoves/movers.py

```diff
@@ -297,6 +306,8 @@ class ScpMover(Mover):
 
     def open_connection(self):
         """Open a connection."""
+        if SSHClient is None or SCPClient is None:
```

Comment:
why not import SSH Client here directly and let it raise the error if not found?

Reply:
(and same for SCPClient where it's used)

Reply:
I thought it better to put these like this so that also the `SCPClient` import was tested already when opening the connection. It is accessed only when `copy()` is called, so the failure can be delayed for a significant time (e.g. with polar satellite data) after the startup. *BUT* now that I read the code, the call to `copy()` actually has the connection creation as the first step, so this will happen in any case.

I'll move the imports as suggested.

Reply:
Done. Needed to add some mocking in the tests.